configure.ac

```diff
@@ -1,8 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 #AC_PREREQ([2.62])
 
-AC_INIT(mono, [5.11.0],
-        [http://bugzilla.xamarin.com/enter_bug.cgi?classification=Mono])
+AC_INIT(mono, [5.11.0], [https://github.com/mono/mono/issues/new])
```

Comment:
could you please leave it on a separate line? we have some scripts that grep the version number out of configure.ac all over the place and I'm not 100% sure all of them handle this :D

Reply:
damn 😄